src/components/Carousel/Carousel.fixture.tsx

```diff
@@ -66,4 +67,23 @@ export default {
       />
     </div>
   ),
+  program: (
+    <div className="container grid grid-cols-12">
+      <Carousel
+        className="col-span-12"
+        visibleItemCount={2}
+        isBlur={false}
+        block={{
+          type: 'Carousel',
+          blocks: new Array(6)
+            .fill({
+              type: 'CarouselProgramsCard',
+              content: BUSINESS_CARD,
+            })
+            .map((_, i) => (i === 1 ? { ..._, content: BUSINESS_CARD } : _)),
```

Comment:
Внутри `map` в `content` подставляется та же константа

```suggestion
          blocks: new Array(6)
            .fill({
              type: 'CarouselProgramsCard',
              content: BUSINESS_CARD,
            })
```